src/effects/native/reverbeffect.cpp

```diff
@@ -26,7 +26,7 @@ EffectManifest ReverbEffect::getManifest() {
     EffectManifestParameter* time = manifest.addParameter();
     time->setId("bandwidth");
     time->setName(QObject::tr("Bandwidth"));
-    time->setName(QObject::tr("Band"));
+    time->setShortName(QObject::tr("BW"));
```

Comment:
Not really a common abbreviation. We cant expect translators to guess what this means.
Up to 11 characters are used for short name ``Moog Filter``,  can`t we use just ``Bandwidth`` then?

The transifex translation service allows to set a character limit. We can use that, and/or add a comment for translator in the code, hinting that they have $ characters max. and should use an abbreviation where appropriate.